database/engine/README.md

```diff
@@ -112,5 +112,34 @@ The Database Engine may keep a **significant** amount of files open per instance
 slave or master server). When configuring your system you should make sure there are at least 60
 file descriptors available per `dbengine` instance.
 
+Netdata allocates 50% of available file descriptors to its web server sockets, 25% to its statsd sockets
+and the rest is available to the daemon and the dbengine. This means that only 25% of the allocated file
+descriptors are accessible by the dbengine. You should take that into account when configuring your service
+or global file descriptor limits.
+
+If for example one wants to allocate 65536 file descriptors to the netdata service on a systemd system
+she needs to edit the netdata systemd file `/etc/systemd/system/multi-user.target.wants/netdata.service`
```

Comment:
she? :woman:

Reply:
We have to respect our female users as well :D

Reply:
`Did You Just Assume My Gender?` meme.jpg :girl:

Reply:
The correct grammar is "If one ... then one ...". We don't use this form though, we just say "you". You really don't want get into gender-specific pronouns, especially nowadays.